protobuf/lib/src/protobuf/json/json.dart

```diff
@@ -2,9 +2,17 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-part of 'internal.dart';
+import 'dart:convert' show base64Decode, base64Encode;
 
-Map<String, dynamic> _writeToJsonMap(FieldSet fs) {
+import 'package:fixnum/fixnum.dart' show Int64;
+
+import '../consts.dart';
+import '../internal.dart';
+import '../utils.dart';
+
+export 'json_vm.dart' if (dart.library.html) 'json_web.dart';
```

Comment:
Iff this uses the vm impl on the VM and when compiled with dart2wasm, and the web impl for dart2js, you might add a comment to that effect.